register.php

```diff
@@ -0,0 +1,942 @@
+<?php
+include "include/header.php"; ?>
+
+<style>
+
+
+.appnitro li
```

Comment:
Styles bitte in eine extra css Datei. Und werden hier wirklich soviele neue styles gebraucht?